x/permission/keeper/keeper.go

```diff
@@ -4,16 +4,15 @@ import (
 	"fmt"
 
 	"cosmossdk.io/math"
+	"github.com/bnb-chain/greenfield/internal/sequence"
```

Comment:
fix the imports

Reply:
Fixed. Use `make format` to format all the code.

Reply:
it seems not...

Reply:
Fixed.